docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
   db:
     image: "mysql:5"
     environment:
+      XWIKI_VERSION: 7.4.3
+      MYSQL_DRIVER_VERSION: 5.1.38
```

Comment:
Considering this remark, indeed, it would look weird that it works.  The reason is that it's also define in the `Dockerfile` (default value). But you're right, it should be inside the other section.  Working on it.